scripts/geonode-bootstrap.py

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Please remove this file from the PR

It is not relevant to the current work, as we are not using GeoNode's internal API here